kaggle_environments/static/player.html

```diff
@@ -500,6 +501,20 @@
           }
         };
 
+        // These are bindings to the 0-9 keys and are milliseconds of timeout per step
+        const speeds = [
+          0,
```

Comment:
IIUC, these values are actually the inverse of speed, so lower is faster?  If so, then it seems a bit weird for 0 to be the fastest and 9 to be the slowest.  Also it might make sense to just remove 0 altogether since it's after 9 on a keyboard?

Reply:
ah yeah I'll invert everything except 0, good point, the intention was for 0 to be "render as fast as the browser can"